Makefile.PL

```diff
@@ -1703,20 +1703,25 @@ sub get_client_version {
     }
 
     if (!$client_version_full) {
-	$client_version_full = "8.0.0.0";
-        print qq{
-WARNING: I could not determine Oracle client version so I\'ll just
-default to version $client_version_full. Some features of DBD::Oracle may not work.
-Oracle version based logic in Makefile.PL may produce erroneous results.
-You can use "perl Makefile.PL -V X.Y.Z" to specify a your client version.\n
+
+      # set a supported client version as default
+      $client_version_full = "9.2.0.4.0";
```

Comment:
I actually dug through the Makefile.PL to determine what version checks were being performed. This version string should meet compatibility needs while not breaking any legacy build requirements.